### ISSUE TITLE
🐛 Errors when creating Stripe Plans with default subscription settings

### ISSUE THREAD

**Issue by allouis** [TryGhost/Ghost#11928]
~The latest Stripe API has problems with 0 amount plans, and when using newer Stripe accounts this outputs errors into the terminal, as the default values for plan amounts is 0.~

The default value for plans is actually an empty string! Which now causes errors with the API.

We should change the defaults to a number rather than a string - to keep functionality the same we could use `0`

To reproduce:

1. Start fresh Ghost Install
2. Link to stripe (Using Connect or API Keys) (Do NOT set any plan pricing)
3. Check your terminal, should see an error like:
```
[2020-06-22 17:49:23] ERROR Invalid integer: 

Invalid integer: 

Error Code: 
    parameter_invalid_integer

----------------------------------------

Error: Invalid integer: 
    at Function.generate (/home/user/usr/src/ghost.org/Ghost/node_modules/stripe/lib/Error.js:49:16)
    at IncomingMessage.<anonymous> (/home/user/usr/src/ghost.org/Ghost/node_modules/stripe/lib/StripeResource.js:168:39)
    at Object.onceWrapper (events.js:421:28)
    at IncomingMessage.emit (events.js:327:22)
    at IncomingMessage.EventEmitter.emit (domain.js:482:12)
    at endReadableNT (_stream_readable.js:1221:12)
    at processTicksAndRejections (internal/process/task_queues.js:84:21)
```

**Comment by christopherkao** [comment]
<img width="795" alt="Screen Shot 2020-06-22 at 8 25 50 PM" src="https://user-images.githubusercontent.com/5176026/85357643-9f752400-b4c6-11ea-94d3-eaa5ab3bb03e.png">
<img width="785" alt="Screen Shot 2020-06-22 at 8 26 01 PM" src="https://user-images.githubusercontent.com/5176026/85357647-a13ee780-b4c6-11ea-952e-60c069e455c0.png">

I have also seen this error. I am using Ghost latest version and had set my monthly and annual subscriptions to $0 a few days ago. Today I made a Stripe account, hooked up my API keys and saw these errors (screenshots) on my Stripe account.

I noticed the problems when the "Choose this Plan" button had errors on my Ghost website: https://christopherkao.me/signup/.
